src/modules/General/components/Pagination/index.module.scss

```diff
@@ -1,9 +1,27 @@
 @import 'src/styles/constants/_primitives.scss';
 
-.buttonsFont {
-    display: flex;
+@mixin Font {
     font-size: 14px;
```

Comment:
rem

Reply:
Relative font size doesn't affect for font size, border, border radius, line height, ...